### PR TITLE
[MTDSA-5531] - retrieve relief investments RAML

### PR DESCRIPTION
**Pull request by Thepworth95**

### CODE REVIEW

File: resources/public/api/conf/1.0/reliefInvestments_retrieve.raml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+is:
+  - headers.acceptJson
+  - headers.contentTypeJson
+  - headers.testScenarioHeader
+  - errors.formatNino
+  - errors.formatTaxYear
+  - errors.ruleTaxYearRangeInvalid
```
Comment (HarryBaxter) [inline resources/public/api/conf/1.0/reliefInvestments_retrieve.raml:7]: I tried running the RAML locally and it errors saying there is no reference for this. Does this error exist in the errors.raml?

File: resources/public/api/conf/1.0/schemas/retrieveReliefInvestments/retrieve_relief_investments_response.json [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+          "reliefClaimed": {
+            "description": "The amount of relief being claimed for the tax year.",
+            "type": "number",
+            "example": "2000.99"
+          }
+        }
+      }
+    }
+  },
+  "links": {
```
Comment (james-work-account) [inline resources/public/api/conf/1.0/schemas/retrieveReliefInvestments/retrieve_relief_investments_response.json:183]: links are in the wrong place, you also need to specify that the links are required and that the fields inside the links are required (see how it's done in another GET request schema)

File: resources/public/api/conf/1.0/schemas/retrieveReliefInvestments/retrieve_relief_investments_response.json [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+    "type": "array",
+    "items": {
+      "type": "object",
+      "description": "A list of endpoint links that indicate possible actions related to the current resource",
+      "properties": {
+        "href": {
+          "description": "The relative url of the endpoint",
+          "type": "string",
+          "example": "/accounts/self-assessment/{nino}/charges/{transactionId}"
```
Comment (james-work-account) [inline resources/public/api/conf/1.0/schemas/retrieveReliefInvestments/retrieve_relief_investments_response.json:192]: use an example relevant to this endpoint; charges doesn't make sense here

File: resources/public/api/conf/1.0/examples/retrieveReliefInvestments/retrieve_relief_investments_response.json [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+      "uniqueInvestmentRef": "123412/1A",
+      "socialEnterpriseName": "SE Inc",
+      "dateOfInvestment": "2020-12-12",
+      "amountInvested": 123123.22,
+      "reliefClaimed": 3432.00
+    }
+  ],
+  "links":[
+    {
+      "href":"/reliefs/investment/{nino}/{taxYear}",
```
Comment (james-work-account) [inline resources/public/api/conf/1.0/examples/retrieveReliefInvestments/retrieve_relief_investments_response.json:50]: replace `{nino}` and `{taxYear}` with example nino and taxYear (also change this in the other links)

File: resources/public/api/conf/1.0/examples/retrieveReliefInvestments/retrieve_relief_investments_response.json [hunk 5]
```diff
@@ -0,0 +1,65 @@
+{
```
Comment (james-work-account) [inline resources/public/api/conf/1.0/examples/retrieveReliefInvestments/retrieve_relief_investments_response.json:1]: don't know if this needs to be in its own package

File: resources/public/api/conf/1.0/schemas/retrieve_relief_investments_response.json [hunk 6]
```diff
@@ -0,0 +1,220 @@
...
+    "links": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "description": "A list of endpoint links that indicate possible actions related to the current resource",
+        "properties": {
+          "href": {
+            "description": "The relative url of the endpoint",
+            "type": "string",
+            "example": "/individuals/reliefs/investment/{nino}/{taxYear}"
```
Comment (james-work-account) [inline resources/public/api/conf/1.0/schemas/retrieve_relief_investments_response.json:191]: Please replace `{nino}` and `{taxYear}` with valid examples (like you did in the example response JSON)

**Pull request merged**